README.md

```diff
@@ -13,3 +13,25 @@ It is meant to be run regularly by a cronjob.
 ```bash
 paster --plugin=ckanext-stadtzh-theme stadtzhtheme cleanup_datastore -c /etc/ckan/default/development.ini
 ```
+
+## Logic for Autosuggestion
+
+This extensions currently provides one action for collect autosuggestion
+from the solr handler `/suggest`: 
+The action is `ogdzh_autosuggest` with the paramters:
+
+ - `q`: the search term 
+ - `cfq`: the context: possible context are all facet names. They can be added using `AND`.   
+
+Here are some examples for api calls to get the autosuggestions:
+```
+http://stadtzh.lo/api/3/action/ogdzh_autosuggest?q=velo&cfq=Wasser AND csv
```

Comment:
maybe use this as an example instead:
`q=velo&cfq=geodaten%20AND%20csv`
because it is complete and at least right now also gets some results :)